### PR TITLE
Made a layout with stars

### PR DESCRIPTION
**Pull request by ia-lerimund**
- [DEMO LINK](https://ia-lerimund.github.io/layout_stars/)
- [TEST REPORT LINK](https://ia-lerimund.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  padding: 0.5px;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(1n),
+.stars--2 .stars__star:nth-child(1n+3),
+.stars--3 .stars__star:nth-child(1n+4),
+.stars--4 .stars__star:nth-child(1n+5),
+.stars--5 .stars__star:nth-child(1n+6) {
+  background-image: url(./images/star.svg);
```
Comment (DarkMistyRoom) [inline src/style.css:25]: you can do it in the main selector

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 2px;
```
Comment (DarkMistyRoom) [inline src/style.css:15]: image
and better use one-direction margins. for example margin-right: 4px.

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (ia-lerimund) [reply src/style.css]: Thank you! Changed styles but tests failed :( What can be the reason?

**Review by ivanchuk-yurii — APPROVED** [review]
I think you haven't deployed your last changes, but this error in tests is minor. however remember that you should do `npm run deploy` after every changes in pr

**Pull request closed without merging**